docs/tutorials/model_correction.md

```diff
@@ -91,24 +91,32 @@ for diff in validation_result.diffset:
 # print some of the autogenerated template
 for templ in generated_templates.get_templates():
     templ = templ.inline_dependencies()
+    # skipping uninteresting templates for the sake of the demo
```

Comment:
I understand what's happening here, but I wonder if there's a better way to get the same result in this tutorial. I don't want this line to start showing up in people's code who have just copy pasted and not read the comment.